### DISCUSSION TITLE
Newbie ProTools questions

### DISCUSSION THREAD

**Discussion by fojerhar** [apohl79/audiogridder#449] [Help]
Hi
I started to use audiogridder since 1.1.1, as the aax format arrived, because I use protools only.
My setup:
Hackintosh, Catalina 10.15.7
Processor AMD Threadripper 3970X, 32 gigs ram, etc.....
I would like to use it on my local computer, since the pro tools runs up to 100 percent usage I still have 24 unused cores 48 threads to do the job for me.
So I mainly use it for the master bus to use there the big cpu hungry acustica plugins.
Some issues:
-For me it seems that if I turn on the buffer size (even in 1 block) the audio quality changes. Sound becomes soft, and weird things happening with the transients, but maybe it is a delay compensation protools issue. But I m happy with the 0 buffer sound absolutely.
-I made 4 mixes with this on my master bus since the last 1-2 weeks. Yesterday I had to reopen the session and the plugin doesn t find the server. If I click on connect I have a few frames sound and PT freezes with beach ball.
If I open a new plugin on the same bus, everything works fine, and I could re open it today also.
Or if i change the plugin preset the communication works again, but I loose my plugins and settings....
- Tried to make a preset of plugin chain(in audiogridder) in protools but seems it is not working
- If I put on a plugin in one channel (one channel test session) it charges both PT and AG with cpu usage, so I do not win procession power in PT (but with AG on masterbus I could load more plugins without errors than PT)
- I cannot copy an inster into other channel

So, I would like to use this for taking out the performance from PT and AG can use ALL my 64 threads for my plugins.
Just let me know, if I'm thinking wrong
On the sceenshot are two master inserts. Uppes last week (cannot connect to server) lower is yesterday saved (works fine) (but it would be nice to have my work back...:) )
In this case I have plugs in PT and AG too
Second screeshot, only one track only one ag plugin and both PT and AG uses the same amound of process, so the plugins using my cpu two times

Thank you for your attention
Have a nice day
Viktor
<img width="1323" alt="Screenshot 2021-03-24 at 12 48 55" src="https://user-images.githubusercontent.com/72139943/112305877-78d75a00-8c9f-11eb-9650-574bd2ce1083.png">

![Screenshot 2021-03-24 at 12 40 27](https://user-images.githubusercontent.com/72139943/112305059-8b04c880-8c9e-11eb-968d-51aa32a9df54.jpg)

**Comment by CaBuffalo** [comment]
By looking at your screenshot, it looks like you are using virtual machine
to host your slave server. If so, it takes your CPU power.

There is a known issue to have Acustica working properly in AG that Andreas
is working on it. It's mainly from the Acustica support from my
understanding.

On Wed, Mar 24, 2021 at 4:51 AM fojerhar ***@***.***> wrote:

> Hi
> I started to use audiogridder since 1.1.1, as the aax format arrived,
> because I use protools only.
> My setup:
> Hackintosh, Catalina 10.15.7
> Processor AMD Threadripper 3970X, 32 gigs ram, etc.....
> I would like to use it on my local computer, since the pro tools runs up
> to 100 percent usage I still have 24 unused cores 48 threads to do the job
> for me.
> So I mainly use it for the master bus to use there the big cpu hungry
> acustica plugins.
> Some issues:
> -For me it seems that if I turn on the buffer size (even in 1 block) the
> audio quality changes. Sound becomes soft, and weird things happening with
> the transients, but maybe it is a delay compensation protools issue. But I
> m happy with the 0 buffer sound absolutely.
> -I made 4 mixes with this on my master bus since the last 1-2 weeks.
> Yesterday I had to reopen the session and the plugin doesn t find the
> server. If I click on connect I have a few frames sound and PT freezes with
> beach ball.
> If I open a new plugin on the same bus, everything works fine, and I could
> re open it today also.
> Or if i change the plugin preset the communication works again, but I
> loose my plugins and settings....
>
>    - Tried to make a preset of plugin chain(in audiogridder) in protools
>    but seems it is not working
>    - If I put on a plugin in one channel (one channel test session) it
>    charges both PT and AG with cpu usage, so I do not win procession power in
>    PT (but with AG on masterbus I could load more plugins without errors than
>    PT)
>    - I cannot copy an inster into other channel
>
> So, I would like to use this for taking out the performance from PT and AG
> can use ALL my 64 threads for my plugins.
> Just let me know, if I'm thinking wrong
> On the sceenshot are two master inserts. Uppes last week (cannot connect
> to server) lower is yesterday saved (works fine) (but it would be nice to
> have my work back...:) )
> In this case I have plugs in PT and AG too
> Second screeshot, only one track only one ag plugin and both PT and AG
> uses the same amound of process, so the plugins using my cpu two times
>
> Thank you for your attention
> Have a nice day
> Viktor
> [image: Screenshot 2021-03-24 at 12 48 55]
> <https://user-images.githubusercontent.com/72139943/112305877-78d75a00-8c9f-11eb-9650-574bd2ce1083.png>
>
> [image: Screenshot 2021-03-24 at 12 40 27]
> <https://user-images.githubusercontent.com/72139943/112305059-8b04c880-8c9e-11eb-968d-51aa32a9df54.jpg>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/449>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C3YRCD7SMD6VMF6HDTFHG5DANCNFSM4ZXDFEVA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by fojerhar** [reply]
Hi

I do not use any virtual things. As i realized th AG address 192.168.10.115 is on the same network where my tp-link usb wifi adaptor is (that is the .114)
But in the plugin I always see two servers
<img width="932" alt="Screenshot 2021-03-24 at 20 47 18" src="https://user-images.githubusercontent.com/72139943/112375550-b52ca980-8ce3-11eb-93dd-fb59f6746b53.png">
And as in the screenshot seen the first two plugins were made last week (not working, cannot connect) and the third one was made now, as I can reopen the session and it runs.

Acustica plugins: all the plugins are working flawlessly from this company and I have a ton of them. Loads faster in AG and reacts faster too than in PT... :) nice job

I try to find out this recall issue, as I try to cnnect to the server beachball comes in PT, and do not generates any crashlogs neither.

**Reply by fojerhar** [reply]
Issue:

As I turn off my tp-link usb wifi dongle in sys pref/network PT freezes.
Reopened PT server was not seen. Restart server, and now I have only one server option.
Turned off airport wifi, PT crashed (with log now finally).

Okay, now realized what is happening. the server uses my tp-link wifi ip address to communicate. If I turn off every network instance in system pref the server is not working.
And in my case my ip address is changing, so I cannot open my sessions wich had a different ip address last week (looks for x.x.x.111.

For my case would be nice to use the 127.0.0.1 port, but how can I resolve the last week sessions?

Issue B? in this case, if I try to open the session in other studio, where of course the ip is different, it also wont work

Thank you

**Comment by CaBuffalo** [comment]
Can you explain more detail about your server/client set up? How did you
set your network?

Thanks.

On Wed, Mar 24, 2021 at 2:52 PM fojerhar ***@***.***> wrote:

> Issue:
>
> As I turn off my tp-link usb wifi dongle in sys pref/network PT freezes.
> Reopened PT server was not seen. Restart server, and now I have only one
> server option.
> Turned off airport wifi, PT crashed (with log now finally).
>
> Okay, now realized what is happening. the server uses my tp-link wifi ip
> address to communicate. If I turn off every network instance in system pref
> the server is not working.
> And in my case my ip address is changing, so I cannot open my sessions
> wich had a different ip address last week (looks for x.x.x.111.
>
> For my case would be nice to use the 127.0.0.1 port, but how can I resolve
> the last week sessions?
>
> Issue B? in this case, if I try to open the session in other studio, where
> of course the ip is different, it also wont work
>
> Thank you
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/449#discussioncomment-527034>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6G6GWZPDP7LAOWOC3DTFJNLXANCNFSM4ZXDFEVA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by fojerhar** [reply]
Hi
There is no setup. The plugin and the server runs on the same machine.
I would like to use this s acustica company made the modula server (and they are working on the network solution since a year, what was solved here with AG... :) ). 
I just want to take out the processing power from protools in the same computer.
And in this case AG is a perfect solution, I can use any plugins and any plugin chains what is the most important, and I have a plenty of free cpu in my computer.

On the pictures above the two servers are my wifi routers, which had any ip address

**Comment by fojerhar** [comment]
Found a solution for some issues:
I have setted the server address to my local ip 127.0.0.1 at server .cfg. In this case if I opened a plugin, it did not found any servers, but i wrote the 127.0.0.1 into, and it is working. This address will not change for me....

Rescueing old session inserts which were made on a different ip address:
turn off my usb wifi, which has an ip address (what AG server wanted to use)
create a new session
import the tracks with AG plugin inserts
close the session
in audiogridderplugin.cfg correct the line "Last server" to 127.0.0.1
open the session
reconnect to the sever, and working
If I re import these corrected track to the old PT session PT freezes. So imported all the other tracks to the new session, and I got back my work... :) happy for that (I just had to correct two places in a whole live concert mix)

And Double thumbs up!!!!
With using the same amount (or more) of acustica plugins in AG on my masterbus offline render now goes with x2.0 speed, if I have regular same plugins inside PT it went with x0.7
